core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/update.kt

```diff
@@ -270,6 +277,7 @@ public fun <T, C> Update<T, C>.with(expression: UpdateExpression<T, C, C?>): Dat
     }
 
 /** [Update with][Update.with] to provide a new value for every selected cell giving its row. */
+@ExcludeFromSources
```

Comment:
What's happened when we exclude this from source? Or all documentation interfaces should be shipped with this annotation?

Reply:
Yes, indeed. We should annotate all our documentation interfaces with `@ExcludeFromSources`. This will exclude them from the sources+compilation and thus shrink the bytecode (in theory). For this PR I started gradually introducing them on a few interfaces already, but far from all of them (https://github.com/Kotlin/dataframe/issues/605). There's no rush though, so we can just annotate them whenever we like.